depends/packages/qt.mk

```diff
@@ -87,7 +87,6 @@ $(package)_config_opts += -no-feature-printdialog
 $(package)_config_opts += -no-feature-printer
 $(package)_config_opts += -no-feature-printpreviewdialog
 $(package)_config_opts += -no-feature-printpreviewwidget
-$(package)_config_opts += -no-feature-regularexpression
```

Comment:
It seems a bit excessive to add regex support just for this...

Reply:
Would agree here. If you really need complex regexps then I don't mind adding the dep, but it seems overkill for this.

Reply:
OTOH #17213 seems to need qt-regexp as well.

Reply:
Reverted back.